test/org/jetbrains/plugins/scala/codeInsight/intentions/booleans/ReplaceEqualsOrEqualityInMethodCallExprIntentionTest.scala

```diff
@@ -21,8 +21,34 @@ class ReplaceEqualsOrEqualityInMethodCallExprIntentionTest extends ScalaIntentio
 
   def testReplaceQuality2() {
     val text = "if (a.eq<caret>uals(false)) return"
-    val resultText = "if (a.<caret>==(false)) return"
+    val resultText = "if (a <caret>== false) return"
 
     doTest(text, resultText)
   }
+
+  def testReplaceQuality2_withSpace() {
+    val text = "if (a.eq<caret>uals( false  )) return"
+    val resultText = "if (a <caret>== false) return"
+
+    doTest(text, resultText)
+  }
+
+  def testReplaceTuple(): Unit = {
+    (true, false).equals(false, false)
+    val text = "(true, false).equ<caret>als  (false, false)"
+    //TODO desired (true, false) <caret>== (false, false)
+    val resultText = "(true, false) <caret>==(false, false)"
+
+    doTest(text, resultText)
+  }
+
+  def testReplaceTuple_2(): Unit = {
+    (true, false).equals(false, false)
+    val text = "if((true, false).equ<caret>als  (false, false)) return "
+    //TODO desired (true, false) <caret>== (false, false)
+    val resultText = "if ((true, false) <caret>==(false, false)) return"
```

Comment:
IDEA reformats changed lines of code after write action. There is a separate setting for that space:
https://youtrack.jetbrains.com/issue/SCL-3849
But probably for operators space shouldn't be removed, I'll fix it.

`commitDocument` method should be called after direct modification of document text to update psi. I think it's not needed here.